dali/util/image.h

```diff
@@ -36,45 +36,88 @@ namespace dali {
 
 class ImgSetDescr {
  public:
-  ~ImgSetDescr()                  { clear(); }
+  ~ImgSetDescr()                                { clear(); }
   inline void clear() {
     for (auto &ptr : data_) delete[] ptr;
-    data_.clear(); sizes_.clear();
+    data_.clear(); shapes_.clear();
```

Comment:
Please, split those statements to separate lines.

Reply:
DONE